data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "ark:/21198/a310jfBd",
+  "type": "Codicological Unit",
+  "label": "Sinai Syriac 2A, ff. 1-152",
+  "unit_start": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "1"
+  },
+  "unit_end": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "152"
+  },
+  "origin": {
+    "date_human-readable": "6th c. CE",
+    "date_normalized": "0500/0600",
+    "place": {
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/place/1",
+      "attested_name_NS": "ܛܘܪ ܣܝܢܝ",
+      "attested_name": "Mount Sinai",
+      "source": "Attested in the colophon on f. 152v"
+      }
+  },
+  "associated_persons":
+    [{
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/person/1",
+      "attested_name_NS": "ܝܥܩܘܒ ܒܪܐ ܕܒܣܝܠ",
+      "attested_name": "Ya'qub bar d-Basil",
+      "souce": "Attested in the note on f. 114v"
+    },
+    {
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/person/2",
+      "attested_name_NS": "ܒܪ ܫܒܘܢ ܒܪܐ ܕܒܣܝܠ",
+      "attested_name": "Bar-Shabon bar d-Basil",
+      "souce": "Attested in the note on f. 114v"
+    }],
+  "associated_places":
+  [{
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/place/2",
+      "attested_name_NS": "ܐܝܘ ܩܪܝܬܐ",
+      "attested_name": "The Village Yw",
+      "source": "Provided in connection with Jacob and Bar-Shabbon in the note on f. 114v"
+      }],
+    "language": ["Syriac"],
+    "writing_and_hands": [
+      {
+        "writing_system": ["Syriac"],
+        "script": ["Estrangela"],
+        "ink_color": "black",
+        "hand_note": ["Taw, Alaph, and Dalath/Resh all exhibit features of Serto."]
+      }],
+  "support": "Parchment",
+  "condition_note": "Quire signatures at beginning and end of most quires. Quire 20 is paper.",
```

Comment:
Do we need a separate field with boolean or cv for quire signatures?

Reply:
Also, do we need a separate field for other ordering systems like catchwords? Do we need to provide more detail than just 'it has quire signatures', e.g. the writing system used, the numerals used (cardinal vs ordinal), if it's numerals or written out words, where on the page they tend to appear (and whether they appear at the beginning and ending of the quire), etc. Whether the signatures are by the same hand?

Reply:
Quire signatures would be in the collation note. Support note for Quire 20?